### DISCUSSION TITLE
How to change color of Highlight?

### DISCUSSION THREAD

**Discussion by rvalitov** [taiga-family/taiga-ui#919] [Q&A] [answered]
Hi!
I want to change color of the Highlight https://taiga-ui.dev/directives/highlight
Is it possible?

**Comment by waterplea** [comment, marked as answer]
It uses the same CSS variable as native selection: `--tui-selection`. It's not configurable at the moment. I think you can add `tuiHighlightColor` input defaulting to that variable. With a PR I mean :)
